docs/detections/prebuilt-rules/rule-details/aws-security-group-configuration-change-detection.asciidoc

```diff
@@ -0,0 +1,84 @@
+[[aws-security-group-configuration-change-detection]]
+=== AWS Security Group Configuration Change Detection
+
+Identifies a change to an AWS Security Group Configuration. A security group is like a virtul firewall and modifying configurations may allow unauthorized access. Threat actors may abuse this to establish persistence, exfiltrate data, or pivot in a AWS environment.
+
+*Rule type*: query
+
+*Rule indices*:
+
+* filebeat-*
+* logs-aws*
+
+*Severity*: low
+
+*Risk score*: 21
+
+*Runs every*: 10 minutes
+
+*Searches indices from*: now-30m ({ref}/common-options.html#date-math[Date Math format], see also <<rule-schedule, `Additional look-back time`>>)
+
+*Maximum alerts per execution*: 100
+
+*References*:
+
+* https://docs.aws.amazon.com/AWSEC2/latest/WindowsGuide/ec2-security-groups.html
+
+*Tags*:
+
+* Elastic
+* Cloud
+* AWS
+* Continuous Monitoring
+* SecOps
+* Network Security
+
+*Version*: 1
+
+*Added ({stack} release)*: 7.15.0
+
+*Rule authors*: Elastic, Austin Songer
+
+*Rule license*: Elastic License v2
+
+==== Potential false positives
+
+A security group may be created by a system or network administrator. Verify whether the user identity, user agent, and/or hostname should be making changes in your environment. Security group creations from unfamiliar users or hosts should be investigated. If known behavior is causing false positives, it can be exempted from the rule.
+
+==== Investigation guide
+
+
+[source,markdown]
+----------------------------------
+## Config
+
+The AWS Fleet integration, Filebeat module, or similarly structured data is required to be compatible with this rule.
+----------------------------------
+
+
+==== Rule query
+
+
+[source,js]
+----------------------------------
+event.dataset:aws.cloudtrail and event.provider:iam.amazonaws.com and
+event.action:(AuthorizeSecurityGroupEgress or CreateSecurityGroup or
+ModifyInstanceAttribute or ModifySecurityGroupRules or
+RevokeSecurityGroupEgress or RevokeSecurityGroupIngress) and
+event.outcome:success
+----------------------------------
+
+==== Threat mapping
+
+*Framework*: MITRE ATT&CK^TM^
+
+* Tactic:
+** Name: Persistence
+** ID: TA0003
+** Reference URL: https://attack.mitre.org/tactics/TA0003/
+
+
+* Tactic:
+** Name: Defense Evasion
+** ID: TA0005
+** Reference URL: https://attack.mitre.org/tactics/TA0005/
```

Comment:
Is this missing a technique?

Reply:
There is nothing defined in the [rule](https://github.com/elastic/detection-rules/blob/main/rules/integrations/aws/persistence_ec2_security_group_configuration_change_detection.toml).

Tactic(s) are required but not techniques or sub